pyproject.toml

```diff
@@ -216,6 +216,20 @@ splinter = "~0.21.0"
 # Serialization of tracebacks - used when running unittest with --parallel flag
 tblib = "~3.0.0"
 
+
+[tool.poetry.group.observability.dependencies]
+opentelemetry-api = "^1.22.0"
+opentelemetry-sdk = "^1.22.0"
+opentelemetry-exporter-otlp = "^1.22.0"
+opentelemetry-instrumentation-django = "^0.43b0"
+opentelemetry-instrumentation-celery = "^0.43b0"
+opentelemetry-instrumentation-psycopg2 = "^0.43b0"
+opentelemetry-instrumentation-dbapi = "^0.43b0"
+opentelemetry-instrumentation-redis = "^0.43b0"
+opentelemetry-instrumentation-logging = "^0.43b0"
+opentelemetry-instrumentation-mysqlclient = "^0.43b0"
+opentelemetry-instrumentation-wsgi = "^0.43b0"
```

Comment:
Are these actually development-only dependencies? I would have thought they'd need to be added to the main project dependencies instead of in this group.

Reply:
I updated it to make it a requirement. That got missed since the docker container includes everything for dev